strr-examiner-web/app/composables/useExaminerRoute.ts

```diff
@@ -51,10 +60,30 @@ export const useExaminerRoute = () => {
       )
 
       if (examinerActions && examinerActions.length > 0) {
-        const leftButtons: ConnectBtnControlItem[] = []
-        const rightButtons: ConnectBtnControlItem[] = []
+        const currentButtons = mergeWithExisting ? getButtonControl() : { leftButtons: [], rightButtons: [] }
```

Comment:
Maybe you can walk me thru this piece.

Reply:
Sure, this has been updated in [commit](https://github.com/bcgov/STRR/pull/669/commits/4610489df2e80ae28acc88c215c752ef38cd4d6d) and cleaned up.